source/backend/api/Helpers/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -178,6 +178,15 @@ private async Task HandleExceptionAsync(HttpContext context, Exception ex)
 
                 _logger.LogError(ex, "Contractor User missing as a team member on the creation of Acquisition File");
             }
+            else if (ex is ExportHasNoDataException)
+            {
+                var exception = ex as ExportHasNoDataException;
+                code = HttpStatusCode.Conflict;
```

Comment:
would a 404 make more sense? that is what we throw when a user requests an id that does not exist.

Reply:
I also question the need for another exception. To me 404 would be sufficient. Or another option I am pondering on my PR is to use a non-error status code **204 No Content** - because in theory the API shouldn't be concerned that the frontend wants to display a warning if no results are found. 
In my opinion, the API should not return an error code because no error happened, it just didn't find any matching records. 

What do you think of using 204 instead of 400/404 here @eddherrera and @devinleighsmith ?